Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -426,6 +416,20 @@ public class NavigationViewportDataSource: ViewportDataSource {
             overviewMobileCamera.bearing = !isWalking ? bearing : headingDirection
             overviewCarPlayCamera.bearing = bearing
         }
+
+        if overviewCameraOptions.zoomUpdatesAllowed || overviewMobileCamera.zoom == nil {
+            overviewMobileCamera.zoom = overviewCameraZoom(remainingCoordinatesOnRoute,
+                                                           pitch: overviewMobileCamera.pitch,
+                                                           bearing: overviewMobileCamera.bearing,
+                                                           edgeInsets: viewportPadding,
+                                                           maxZoomLevel: overviewCameraOptions.maximumZoomLevel)
+
+            // NOTE: zoom method adds some extra padding to the viewport. We need this extra padding in CarPlay
+            // to avoid overlap of the route, street name labels, and control buttons.
+            overviewCarPlayCamera.zoom = zoom(remainingCoordinatesOnRoute,
+                                              edgeInsets: carPlayCameraPadding,
+                                              maxZoomLevel: overviewCameraOptions.maximumZoomLevel)
```

Comment:
I wonder if we still can use `overviewCameraZoom`, but with additional `carPlayCameraPadding`?

Reply:
I hardcoded this additional padding and switched to `overviewCameraZoom` in CarPlay